Modules/Microsoft365DSC/DSCResources/MSFT_EXORecipientPermission/MSFT_EXORecipientPermission.schema.mof

```diff
@@ -0,0 +1,16 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXORecipientPermission")]
+class MSFT_EXORecipientPermission : OMI_BaseResource
+{
+    [Key, Description("The mailbox the permission should be given on.")] String Identity;
+    [Key, Description("The account to give the permission to.")] String Trustee;
```

Comment:
Would it be an idea to make this parameter a collection? That way we can add one resource per Identity to a config instead of one resource per Trustee per Identity. That would dramatically reduce the number of resources required in the config to configure a larger number of Trustees.